blocks/library/embed/index.js

```diff
@@ -1,7 +1,9 @@
 /**
  * WordPress dependencies
  */
-import { Button, Placeholder } from 'components';
+import Button from 'components/button';
+import Placeholder from 'components/placeholder';
+import HtmlEmbed from 'components/html-embed';
```

Comment:
Copying from slack: Try changing this `import { Dashicon } from 'components';` in the placeholder component to `import { Dashicon } from '../';`